awx/ui_next/src/components/JobList/JobListItem.jsx

```diff
@@ -91,18 +91,22 @@ function JobListItem({
           >
             {job.status === 'failed' && job.type === 'job' ? (
               <LaunchButton resource={job}>
-                {({ handleRelaunch }) => (
-                  <ReLaunchDropDown handleRelaunch={handleRelaunch} />
+                {({ handleRelaunch, isSending }) => (
+                  <ReLaunchDropDown
+                    handleRelaunch={handleRelaunch}
+                    isSending={isSending}
+                  />
                 )}
               </LaunchButton>
             ) : (
               <LaunchButton resource={job}>
-                {({ handleRelaunch }) => (
+                {({ handleRelaunch, isSending }) => (
```

Comment:
can we rename this is isLaunching? I think that more clear about what's happening